### DISCUSSION TITLE
pag文件在部分机型动画不循环的问题

### DISCUSSION THREAD

**Discussion by Xiaolianmao** [Tencent/libpag#1745] [Q&A] [answered]
环境:
macOS Ventura13.4 | AE  23.4.0(Build 53) | PAGViewer 4.3.3.94(beta)

问题:
pag文件部分机型动画不循环,需要手动切换页面才恢复
    典型机型:
    Google Pixel 4、VIvo x23

文件:
[问题文件.zip](https://github.com/Tencent/libpag/files/12710426/default.zip)

**Comment by domchen** [comment, marked as answer]
看下是不是自己设置了repeatCount，改成0或者负值才是无限循环。还有问题请构造一个复现的demo上传。
